test-cases/longevity/longevity-lwt-4days-multidc.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 5760
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml n=1100200300 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50" ]
```

Comment:
You need to set the NetworkReplicationFactor so it will work.

Reply:
fixed.

We can't use '-schema' option for user profile cmdline, so I readded a new profile for multi-dc and assign replicator parameter inside it.